server/src/main/java/org/opensearch/action/admin/cluster/node/stats/NodeStats.java

```diff
@@ -156,6 +160,11 @@ public NodeStats(StreamInput in) throws IOException {
             shardIndexingPressureStats = null;
         }
 
+        if (in.getVersion().onOrAfter(Version.V_3_0_0)) {
+            searchBackpressureStats = in.readOptionalWriteable(SearchBackpressureStats::new);
```

Comment:
for 3.0 should this be optional?

Reply:
All fields of `NodeStats` are read using `readOptionalWriteable`. This is because fields are only returned when they are explicitly contained in the list of requested metrics. This may be done to reduce the verbosity of node stats if all fields were returned.

Request: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/action/admin/cluster/node/stats/TransportNodesStatsAction.java#L102-L123

Service: https://github.com/opensearch-project/OpenSearch/blob/main/server/src/main/java/org/opensearch/node/NodeService.java#L180-L199